src/openapi/spec/register-client-schema.ts

```diff
@@ -0,0 +1,39 @@
+import { createSchemaObject, CreateSchemaType } from '../openapi-types';
+
+export const schema = {
+    type: 'object',
+    required: ['appName', 'interval', 'started', 'strategies'],
+    properties: {
+        appName: {
+            type: 'string',
+        },
+        instanceId: {
+            type: 'string',
+        },
+        sdkVersion: {
+            type: 'string',
```

Comment:
We _could_ use [an OpenAPI `pattern`](https://swagger.io/docs/specification/data-models/data-types/#pattern) here to validate that it's valid SemVer, but that might be overkill 🤷🏼

Reply:
Would be nice, but it would make it diverge from the Unleash API, which would be bad. It is a copy paste from https://github.com/Unleash/unleash/blob/main/src/lib/openapi/spec/client-application-schema.ts

Reply:
Got it. If it's a copy-paste, then this is where having those generated types would come in handy, right? Can they generate specs too? If not, find a way to share the schemas? 🤔